### PR TITLE
feat(NotificationBadge): remove isRead prop

### PR DESCRIPTION
**Pull request by gitdallas**
#223

### CODE REVIEW

**Review by thatblindgeye — APPROVED** [review]
🔥

**Review by wise-king-sullyman — APPROVED** [review]
LGTM, just needs conflicts resolved

File: packages/eslint-plugin-pf-codemods/lib/rules/v5/notificationBadge-remove-isRead.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const { getPackageImports } = require('../../helpers');
+
+  // https://github.com/patternfly/patternfly-react/pull/8090
```
Comment (jenny-s51) [inline packages/eslint-plugin-pf-codemods/lib/rules/v5/notificationBadge-remove-isRead.js:3]:
```suggestion
  // https://github.com/patternfly/patternfly-react/pull/8626
```

File: packages/pf-codemods/README.md [hunk 2]
```diff
@@ -322,6 +322,30 @@ We've updated the default value of the `aria-label` attribute for Nav with a `ho
 
 We've update the `aria-label` prop on MenuItemAction, making it required instead of optional.
 
+### notificationBadge-remove-isRead [(#8090)](https://github.com/patternfly/patternfly-react/pull/8090)
```
Comment (jenny-s51) [inline packages/pf-codemods/README.md:325]:
```suggestion
### notificationBadge-remove-isRead [(#8626)](https://github.com/patternfly/patternfly-react/pull/8626)
```

**Review by jenny-s51 — CHANGES_REQUESTED** [review]
Nice @gitdallas , code looks and runs great! 👍 Just needs small update to link to the PR

**Review by jenny-s51 — APPROVED** [review]
👍

**Pull request merged**